src/classroom.rb

```diff
@@ -0,0 +1,17 @@
+class Classrom
+  attr_accessor :label, :students
+
```

Comment:
- [ ] Kindly ensure that the `students `object can only retrieve the value of instance of the variable. Getters method only provide read access to the attributes of an object. ℹ️
```suggestion
  attr_reader :students
```

Reply:
The suggestion is not clear enough. Do you mean to remove access to the "label" attribute as part of the suggestion or to make the "students" array just readable, and let "label" as it is now? Also, I see that you are talking about the "students" object when the marked code is in the "classroom" object code.